foyer/forcefield.py

```diff
@@ -794,6 +795,31 @@ def apply(
             **kwargs,
         )
 
+        # Start storing scaling factors and combining rule to parmed structure
+        # Utilizing parmed gromactop's _Default class
+
+        # Note: nb_func = 1 (LJ) or 2 (Buckingham), foyer forcefield only support 1 at the moment
```

Comment:
Kind of annoying that they handle it by just setting arbitrary integers for the different non-bonded combining rules.  Although I guess that's more on gromacs than on parmed.

Reply:
yeah, ideally, parmed would have a way to store this in their structure, but this default class is specific to their gromacs writer, so everything abide to gromacs annotation.